pipeline/pipeline/argo.libsonnet

```diff
@@ -215,17 +215,23 @@
 
     config: {
       apiVersion: "v1",
-      // The commented out section creates a default artifact repository
-      // This section is not deleted because we might need it in the future.
-      // And it takes time to get this string right.
-      //data: {
-      //  config: "executorImage: argoproj/argoexec:v2.2.0\nartifactRepository:\n s3:
-      //  \n  bucket: mlpipeline\n  endpoint: minio-service.kubeflow:9000\n  insecure: true
-      //  \n  accessKeySecret:\n   name: mlpipeline-minio-artifact\n   key: accesskey\n  secretKeySecret:
-      //  \n   name: mlpipeline-minio-artifact\n   key: secretkey"
-      //},
+      // Configuring the executor version and the default artifact repository
       data: {
-        config: "executorImage: argoproj/argoexec:v2.2.0",
+        config: |||
+          executorImage: argoproj/argoexec:v2.2.0
+          artifactRepository:
+            s3:
+              bucket: mlpipeline
```

Comment:
I don't think assigning mlpipeline as default for argo is an good option, given argo is now pipeline specific. Kubeflow users could use argo for various purpose. 

also, if you make further change to ksonnet config, make sure do it in kubeflow ks registry here first, 
https://github.com/kubeflow/kubeflow/tree/master/kubeflow
before making any change to our repo. the ks registry here is only used by bootstrapper image, which is not officially supported anymore.

Reply:
LOL. We've been debating whether to add this config for Pipelines-only while someone just went and added this configMap for all Argo users. https://github.com/kubeflow/kubeflow/pull/2238

Yes, all Kubeflow Argo installations since January have default artifact storage which is slightly pipeline-specific =)